src/implementation.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include <initializer_list>
 
+#define SIMDJSON_TRY(EXPR) { auto _err = (EXPR); if (_err) { return _err; } }
```

Comment:
We can't do it, but note that in C++17, this is can be written...
```
 if (auto _err = (EXPR); _err) { return _err; }
```

Reply:
Yeah, I keep writing that and getting slapped down by CI :)